tendrl/commons/flows/import_cluster/__init__.py

```diff
@@ -109,6 +112,46 @@ def run(self):
                 _cluster.current_job['status'] = "finished"
                 _cluster.is_managed = "yes"
                 _cluster.save()
+
+                # Creating a Job to do the profiling task. This job is
```

Comment:
`Creating a job to enable/disable profiling based on flag selected while import flow. This job is separate from import flow and not a child flow. This could be tracked independently once import if over.`

Does this make more sense?